src/EA.Iws.Web/Areas/NotificationApplication/Controllers/FacilityController.cs

```diff
@@ -347,7 +347,8 @@ public async Task<ActionResult> Remove(RemoveFacilityViewModel model, bool? back
                     model.NotificationId,
                     User.GetUserId(),
                     NotificationAuditType.Deleted,
-                    NotificationAuditScreenType.RecoverySite);
+                    model.NotificationType == NotificationType.Disposal ? NotificationAuditScreenType.DisposalSite : NotificationAuditScreenType.RecoverySite);
+
```

Comment:
Space between the code and the closing curly bracket which is causing the CI build to fail.